### PR TITLE
Feature/add skel files

### PR DESCRIPTION
**Pull request by thomasriley**
Related to #1 

Added the cookbook generated from chef-partners/cookbook-guide-generator so that we comply with the Chef Partner Cookbook guidelines. 

Much more to figure out but adding the basics for now before proceed to start writing the actual cookbook and we can continue to iterate on this.

### CODE REVIEW

File: .kitchen-docker.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      - RUN yum -y install lsof which systemd-sysv initscripts openssl
+
+- name: fedora-latest
+  driver:
+    image: fedora:latest
+    pid_one_command: /usr/lib/systemd/systemd
+    intermediate_instructions:
+      - RUN dnf -y install yum which systemd-sysv initscripts
+
+- name: ubuntu-12.04
```
Comment (majormoses) [inline .kitchen-docker.yml]: We should not put any effort into supporting 12.04 as it is EOL: https://wiki.ubuntu.com/Releases

File: .kitchen-docker.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  name: dokken
+
+provisioner:
+  name: dokken
+
+verifier:
+  name: inspec
+
+platforms:
+- name: debian-7
```
Comment (majormoses) [inline .kitchen-docker.yml]: I do not think we should put any effort into debian-7 as it is EOL, I think if anything we should support 8 and 9
https://www.debian.org/releases/

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Sensu Go cookbook changelog
+
+This file is used to track changes made in recent versions of the Sensu Go
+cookbook. 
+
+## [Unreleased]
```
Comment (majormoses) [inline CHANGELOG.md]: We should still follow our changelog conventions with the various headers from plugins. Unless there is a really good reason not to.

Comment (thomasriley) [reply CHANGELOG.md]: Nope, just me copying from the wrong place! :(

**Comment by thomasriley** [comment]
Just pushed up the suggested changes, thanks for the quick review @majormoses and also the Debian knowledge, I don't spend much time working with that I'm mostly using some form of RHEL at work.

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Change Log
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+This CHANGELOG follows the format located [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
+
+## [Unreleased]
+Created repo with initial commit (@mbbroberg)
```
Comment (majormoses) [inline CHANGELOG.md]: these need to go under an `### Added` header

File: CHANGELOG.md (hunk 3 above)
Comment (majormoses) [reply CHANGELOG.md]: what I was getting at was the missing `### Added` header.

File: CHANGELOG.md (hunk 4 above)
Comment (thomasriley) [reply CHANGELOG.md]: D'oh... I should know better by now!
Thanks 👍

**Pull request merged**